fastlane/lib/fastlane/actions/oclint.rb

```diff
@@ -66,7 +66,7 @@ def self.run(params)
         oclint_args << "-enable-clang-static-analyzer" if params[:enable_clang_static_analyzer]
         oclint_args << "-enable-global-analysis" if params[:enable_global_analysis]
         oclint_args << "-allow-duplicated-violations" if params[:allow_duplicated_violations]
-        oclint_args << "-p #{params[:compile_commands]}" if params[:compile_commands]
+        oclint_args << "-p #{File.dirname(params[:compile_commands])}" if params[:compile_commands]
```

Comment:
I've took this from the docs:

> -p <build-path>
> Choose the build directory in which a file named compile_commands.json exists. When no build path is specified, > a search for compile_commands.json will be attempted through all parent paths of the first input file.

http://docs.oclint.org/en/stable/manual/oclint.html#compilation-database

However, I don't know how this translates to this fastlane action. I've been waiting for `scan` support to start using OCLint, so I can't say whether the action is working or not at it is on `master`